flow-typed/npm/babel-core_vx.x.x.js

```diff
@@ -0,0 +1,253 @@
+// flow-typed signature: 58b12a9978527dd32132602d45f30ba4
```

Comment:
Files in the `flow-typed` directory are type definitions for external libraries that will help us in our static type-checking